onedal/common/backend/compute.h

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#ifdef ONEDAL_DATA_PARALLEL
+    #include <CL/sycl.hpp>
+#endif
+
+#ifdef DPCTL_ENABLE

```

Comment:
I'd remove all compile-time dependencies on dpctl.

Reply:
dpctl don't support for Python3.6, 3.8, 3.9. don't compile at all dpcpp part in this case?